test/1_stdlib/RangeTraps.swift

```diff
@@ -30,6 +30,10 @@ import ObjectiveC
 
 var RangeTraps = TestSuite("RangeTraps")
 
+// Don't let the compiler see a constant which would lead to compile errors
+// instead of the traps
+let zero = [0][0]
```

Comment:
Please use `getInt()` from `stdlib/private/StdlibUnittest/OpaqueIdentityFunctions.swift` instead.  (And you can actually inline the call instead of declaring a constant.)